apps/strapi.frameless.io/docs/handboek/strapi-config-import.md

```diff
@@ -0,0 +1,21 @@
+## Strapi Dashboard Configuratie Importeren
+
+⚠️ **Na elke deployment naar productie moet de Strapi configuratie handmatig worden geïmporteerd.**
```

Comment:
Het geldt ook voor de omgevingen (test en acceptatie). Zou je deze ook willen vermelden in de documentatie?